lib/perl/Genome/Model.pm

```
@@ -471,6 +471,8 @@ sub __display_name__ {
 sub create {
     my $class = shift;
 
+    no warnings 'redefine';
+    local *__errors__ = sub { () };
```

What does suppressing errors here do?